easybuild/easyconfigs/c/CRISPR-DAV/CRISPR-DAV-2.3.4-foss-2020b.eb

```diff
@@ -0,0 +1,36 @@
+name = 'CRISPR-DAV'
+version = '2.3.4'
+
+homepage = 'https://github.com/pinetree1/crispr-dav/'
+description = """CRISPR-DAV is a pipeline to analyze
+amplicon-based NGS data of CRISPR clones in a high throughput manner."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/pinetree1/crispr-dav/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['crispr-dav-2.3.4_remove_hardcoding.patch']
+checksums = [
+    '49975cd48bdbf31fe5a9e2aaa3f5ed85d3cc6f65a422ee3aa8daed890159d2ae',  # v2.3.4.tar.gz
+    # crispr-dav-2.3.4-foss2020b_remove_hardcoding.patch
+    '41475a09754d65cc2c88d4161be6eacdfc98242bccbe49778c4f1f74210a6cb2',
+]
+
+# also needs `naturalsort`, which is now an R extension - see https://github.com/easybuilders/easybuild-easyconfigs/pull/13762
+# if missing, try reinstalling R (https://docs.easybuild.io/en/latest/Partial_installations.html#installing-additional-extensions-using-k-skip)
```

Comment:
Not needed here, just mention this in a PR description (not the easyconfig file).

There's probably also other R libraries that are required?

If you want to make a note in the easyconfig, do this, that's sufficient:

```python
dependencies = [
    ('R', '4.0.3'),  # provides naturalsort R package